app/models/webauthn_credential.rb

```diff
@@ -25,13 +25,15 @@ def enable_user_mfa
     return unless user.mfa_device_count_one?
     user.mfa_level = :ui_and_api
     user.mfa_recovery_codes = Array.new(10).map { SecureRandom.hex(6) }
+    user.hashed_mfa_recovery_codes = user.mfa_recovery_codes.map { |code| BCrypt::Password.create(code) }
```

Comment:
As with previous comment, this can be switched to
```ruby
self.hashed_mfa_recovery_codes = Array.new(10).map { BCrypt::Password.create(SecureRandom.hex(6)) }
```
to make the follow up PR a bit more cleaner.

Reply:
I'm intentionally keeping the plain text ones around for the time being, to make sure we are able to roll back if needed

Reply:
Gotcha, makes sense 👍

Reply:
Also i think we can't directly hash them, because then we could never display them for the user to copy!